### ISSUE TITLE
switch to OVN of dev test env? 

### ISSUE THREAD

**Issue by jichenjc** [kubernetes-sigs/cluster-api-provider-openstack#1487]
/kind feature

I am trying to upgrade current devstack from Y to Z now , though stuck https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1398 (happy if someone can provide some tip on why the issue occur)

but I think we should think about move from OVS to OVN in the long run ,as it's by default solution in neutron now for long time
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/hack/ci/cloud-init/controller.yaml.tpl#L85
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/hack/ci/cloud-init/controller.yaml.tpl#L38

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Comment by mdbooth** [comment]
@stephenfin anything you might be able to help with?

**Comment by stephenfin** [comment]
Yeah, sure. I'll take a look. We probably want to aim for Antelope too, if possible. It'll be out soon enough.

**Comment by stephenfin** [comment]
@EmilienM did this in https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1804/ :partying_face:

/close
